source/atlas-cli-admin-api.txt

```diff
@@ -0,0 +1,180 @@
+.. _atlas-cli-admin-api:
+
+===============================================================
+Run {+atlas-cli+} Commands with {+atlas-admin-api+}
+===============================================================
+
+.. meta::
+   :description: Learn to execute Atlas CLI commands using the Atlas Administration API.
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This tutorial shows you how to run {+atlas-cli+} commands with the
+{+atlas-admin-api+}. You can use every {+atlas-admin-api+} resource
+and endpoint from the {+atlas-cli+} to create and manage |service|
+deployments. {+atlas-cli+} support for the {+atlas-admin-api+} provides the
+following benefits:  
+
+- Full feature parity with the {+atlas-admin-api+}
+- Quicker access to new {+atlas-admin-api+} resources and endpoints
+- A unified, predictable command structure for automation
+- Ability to pin a desired API version, ensuring your scripts remain
+  reliable, even if you update the CLI.
+
+Syntax 
+------
+
+To use {+atlas-cli+} with the {+atlas-admin-api+}, run the command in
+the following format: 
+
+.. code-block:: shell 
+
+   atlas api <tag> <operationId> [options]|--file <fileName>.json --version <api-resource-version>
+
+Arguments 
+~~~~~~~~~
+
+.. list-table:: 
+   :widths: 20 10 70 
+   :header-rows: 1
+
+   * - Argument 
+     - Necessity
+     - Description
+
+   * - ``<tag>``
+     - Required
+     - The name of the tag used in the {+atlas-admin-api+} documentation
+       |url| for the |api| resource. The tag is hyphen-separated in the
+       {+atlas-admin-api+} documentation |url|. However, you must
+       convert it to camelcase in the {+atlas-cli+} command syntax. 
+
+       .. code-block:: shell 
+
+          https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/<Tag-Name>/
+       
+       For accessing the resource in the preceding |url|, replace
+       ``<tag>`` with ``tagName`` in the command: 
+
+       .. code-block:: shell 
+
+          atlas admin tagName <operationId> 
```

Comment:
This should be `atlas api tagName <operationId>`.

For consistency, it would maybe be better to use `<>` around `tagName` too.

Like this: `atlas api <tagName> <operationId>`

Reply:
This is an example. Fixed.